pxemgr/pxemanager.go

```diff
@@ -301,7 +335,23 @@ func (mgr *pxeManagerT) Start() error {
 		return err
 	}
 
-	return mgr.startIPXEserver()
+	go func() {
+		err := mgr.startIPXEserver()
+		if err != nil {
+			panic(err)
+		}
+	}()
+
+	go func() {
+		err := mgr.startAPIserver()
+		if err != nil {
+			panic(err)
+		}
+	}()
+
+	select {}
```

Comment:
I thought you wanted to do this in `main.go` since this is the program endpoint. Errors should be propagated back to the caller if possible. In case you really want to do this here there is no need to have an error returned at all, because this will never happen. Then the interface of the method should adopt.